server/assessment/service/exportService.js

```diff
@@ -20,6 +20,43 @@ const EXPORT_TYPE = {
   CSV: 'csv',
 }
 
+// This is used in Global => navigation => Bulk Download
+const exportPublicData = async () => {
```

Comment:
why duplicate this method? wouldn't it be more effective to add some condition to the existing exportData method ?